src/modules/dashboard/components/PermissionManagementDialog/PermissionManagementCheckbox.tsx

```diff
@@ -1,59 +1,72 @@
 import React, { useMemo } from 'react';
 import { defineMessages, FormattedMessage, useIntl } from 'react-intl';
-import { ColonyRole } from '@colony/colony-js';
+import { ColonyRole, ROOT_DOMAIN_ID } from '@colony/colony-js';
 
 import { Checkbox } from '~core/Fields';
-import Heading from '~core/Heading';
 import Popover from '~core/Popover';
+import PermissionsLabel from '~core/PermissionsLabel';
 
-import styles from './PermissionCheckbox.css';
+import styles from './PermissionManagementCheckbox.css';
+
+const idBase =
+  'dashboard.PermissionManagementDialog.PermissionManagementCheckbox';
 
 const MSG = defineMessages({
   roleWithAsterisk: {
-    id: 'admin.ColonyPermissionEditDialog.roleWithAsterisk',
+    id: `${idBase}.roleWithAsterisk`,
     defaultMessage: '{role}{asterisk, select, true {*} false {} }',
   },
   roleDescription0: {
-    id: 'admin.ColonyPermissionEditDialog.roleDescriptionRecovery',
-    defaultMessage: 'Put the Colony into recovery mode.',
+    id: `${idBase}.roleDescriptionRecovery`,
```

Comment:
The id of the message descriptor **must** match the `MSG` object prop. So change either one, but they must match.
```
roleDescriptionRecovery: {
  id: `dashboard.PermissionManagementDialog.PermissionManagementCheckbox.roleDescriptionRecovery`,
  ...
}
```